### PR TITLE
Remove SCS from navbar, add Publications

### PR DESCRIPTION
**Pull request by eddie-knight**
Part of #1257 

[Preview](https://deploy-preview-1265--tag-security.netlify.app)

### CODE REVIEW

**Review by brandtkeller — COMMENTED** [review]
Navbar and the contents of the `publications` page look to be resolved - but any link to markdown file (directly in the repo) is having the `.md` stripped and resulting in a 404.

**Comment by eddie-knight** [comment]
> Navbar and the contents of the `publications` page look to be resolved - but any link to markdown file (directly in the repo) is having the `.md` stripped and resulting in a 404.

Got it— this is clashing with another patch that we put in to remove the `.md` from local links. I'll investigate.

**Comment by mrcdb** [comment]
Related to this, I just noticed PUBLICATIONS.md does not include the links to all the latest CNS WP v2 (e.g. Italian): https://deploy-preview-1265--tag-security.netlify.app/publications/#cloud-native-security-whitepaper

Updating the PUBLICATIONS.md file with the new links may be an additional action item in https://github.com/cncf/tag-security/issues/1257 once this is merged.

**Comment by eddie-knight** [comment]
Good shout @mrcdb. @anvega, can you confirm that you'll be able to help update the publications file? Should we track that as a new gh issue?

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ There is a growing ecosystem of tools that promises to unlock developer producti
 ## Publications
 
 TAG Security has published several resources for the community, which can be
-found in the [publications](PUBLICATIONS.md) document.
+found in the [publications](publications/README.md) document.
```
[the remark this answers is not in the document]
Comment (eddie-knight) [reply README.md:44]: Not sure. Got an alternate phrasing?

[the remark this answers is not in the document]
Comment (brandtkeller) [reply README.md:44]:
```suggestion
found under [publications](publications/README.md).
```

**Review by brandtkeller — APPROVED** [review]
Given scope is not to the content of the publications markdown itself - the current content is resolving links as expected with navbar and link support.

**Review by anvega — APPROVED** [review]

**Pull request merged**